daisy_workflows/export/disk_export_ext.wf.json

```diff
@@ -32,37 +32,50 @@
     }
   },
   "Sources": {
-    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh"
+    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh",
+    "disk_resizing_mon.sh": "./disk_resizing_mon.sh"
```

Comment:
Prepend ${NAME} for this script as well, it makes this source file unique so you can call this workflow multiple times in the same context.